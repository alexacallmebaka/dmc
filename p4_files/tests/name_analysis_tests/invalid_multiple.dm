main : () int {
  a : bool;
  a = true;
  a : void;
  return a;
}