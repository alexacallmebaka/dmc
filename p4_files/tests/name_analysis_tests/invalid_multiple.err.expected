FATAL [4,3]-[4,4]: Invalid type in declaration
FATAL [4,3]-[4,4]: Multiply declared identifier
Name Analysis Failed